frontend/src/components/Forms/ShoutoutsPage/ShoutoutForm.tsx

```diff
@@ -90,24 +84,23 @@ const ShoutoutForm: React.FC = () => {
 
         <Checkbox
           label={{ children: 'Anonymous?' }}
-          style={{ paddingLeft: '2em' }}
+          className={styles.isAnonCheckbox}
           onChange={() => setIsAnon(!isAnon)}
         />
       </div>
 
-      <div style={{ padding: '0.8em 0' }}>
+      <div className={styles.reasonContainer}>
         <Form.Input
           label="Why are they awesome?"
           name="message"
           value={message}
           control={TextArea}
           onChange={(event) => setMessage(event.target.value)}
-          style={{ minHeight: '25vh' }}
           required
         />
       </div>
 
-      <Form.Button floated="right" onClick={giveShoutout}>
+      <Form.Button floated="right" onClick={giveShoutout} style={{ marginBottom: 0 }}>
```

Comment:
Should we define this style in the CSS file too?

Reply:
> Just tested this out. Maybe it should go in a separate PR but the anonymous feature has a bug. If you try submitting a shoutout but check the "Anonymous" checkbox, the shoutout will show up as going to you instead of the person you're trying to submit the shoutout to.

Good catch, I think it's best to handle this in a separate PR. I'll create a new notion issue for this.